### PR TITLE
Add support for the Webhook Endpoint resource

### PR DESCRIPTION
**Pull request by remi-stripe**
cc @stripe/api-libraries

### CODE REVIEW

File: src/StripeTests/Services/WebhookEndpoints/WebhookEndpointServiceTest.cs [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+        public WebhookEndpointServiceTest()
+        {
+            this.service = new WebhookEndpointService();
+
+            this.createOptions = new WebhookEndpointCreateOptions()
+            {
+                EnabledEvents = new string[]
+                {
+                    "*",
```
Comment (ob-stripe) [inline src/StripeTests/Services/WebhookEndpoints/WebhookEndpointServiceTest.cs]: Can you update the test to use `charge.succeeded` instead?

File: src/StripeTests/Services/WebhookEndpoints/WebhookEndpointServiceTest.cs [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+                },
+                Url = "https://stripe.com",
+            };
+
+            this.updateOptions = new WebhookEndpointUpdateOptions()
+            {
+                EnabledEvents = new string[]
+                {
+                    "*",
```
Comment (ob-stripe) [inline src/StripeTests/Services/WebhookEndpoints/WebhookEndpointServiceTest.cs]: Same here.

**Comment by remi-stripe** [comment]
@ob-stripe PTAL

**Pull request merged**